topology-poc.yaml

```diff
@@ -0,0 +1 @@
+nodes:
```

Comment:
we shouldn't need a topology for poc. Topology files are for cardano-node which this doesn't need.